altair/vegalite/v2/examples/histogram_with_a_global_mean_overlay.py

```diff
@@ -7,15 +7,17 @@
 import altair as alt
 from vega_datasets import data
 
-source = data.movies.url
+source = data.movies()
```

Comment:
We should use the URL, since this data is not cached locally in ``vega_datasets`` (and it will cause tests to fail as a result of poor web connectivity)